sanjayypatel/Cogently
# Feature/document upload

Closes #15 Create Document Model

Document model stores uploaded pdf files and associates with an organization and user.

To test:

1) rake db:migrate to create Document table and add organization and user references to Document table.
2) login as first@example.com, pw: helloworld
3) In top left, click "Docs"
4) The Documents#index page is loaded.  Click "New Doc" in right-hand sidebar.
5) Fill out a document name and select a test file to upload and hit "Upload"
6) Should be taken to the Documents#index page, with uploaded document listed. Click "View" next to document name.
7) Taken to Documents#show page.  Click "View Original" to load original doc in browser.
